### ISSUE TITLE
When switching to a fork, the update message is wrong

### ISSUE THREAD

**Issue by damienalexandre** [pyrech/composer-changelogs#18]
Switching to a fork of FOSElasticaBundle provided the wrong message.

I was using `"friendsofsymfony/elastica-bundle": "~3.1"`, and then added:

```
        {
            "type": "vcs",
            "url": "https://github.com/damienalexandre/FOSElasticaBundle"
        }
```

I'm now using the fork, with a specific branch:

```
"friendsofsymfony/elastica-bundle": "dev-feature/dynamic as 3.1"
```

## Expected result

The plugin should show a "accros fork" comparison url: https://github.com/FriendsOfSymfony/FOSElasticaBundle/compare/v3.1.5...damienalexandre:feature/dynamic

Notice the absence of "dev-" in the branch name too.

## Actual result

```
Changelogs summary:

 - friendsofsymfony/elastica-bundle updated from v3.1.5 to dev-feature/dynamic
   See changes: https://github.com/FriendsOfSymfony/FOSElasticaBundle/compare/v3.1.5...dev-feature/dynamic
   Release notes: https://github.com/FriendsOfSymfony/FOSElasticaBundle/releases/tag/dev-feature/dynamic
```

**Comment by pyrech** [comment]
Your issue should be fixed with the PR #19. Thanks for reporting this.

**Comment by damienalexandre** [comment]
:+1: awesome dude :)
